newIDE/app/src/SceneEditor/SetupGridDialog.js

```diff
@@ -69,7 +71,7 @@ export default class SetupGridDialog extends Component {
               type="number"
               value={this.state.gridWidth}
               onChange={(e, value) =>
-                this.setState({ gridWidth: parseInt(value, 10) })
+                this.setState({ gridWidth: parseFloat(value, 10) })
```

Comment:
parseFloat is taking just the value argument, you can remove 10 which is unused.